src/apps/sorting-visualizer/components/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, ReactNode } from 'react';
+import classes from "./tooltip.module.scss";
+
+interface TooltipProps {
+  text: string;
+  children: ReactNode;
+}
+
+const Tooltip: FC<TooltipProps> = ({ text, children }) => {
```

Comment:
We can stick to normal function standards for consistency

https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/